app/scripts/modules/core/widgets/spelText/spelText.decorator.js

```diff
@@ -20,30 +20,34 @@ let decorateFn = function ($delegate, jsonListBuilder, spelAutocomplete) {
       }
 
       // the textcomplete plugin needs input texts to marked as 'contenteditable'
+      let spelEnabled = !el.attr('class').contains('nospel');
```

Comment:
There's a check right above this line to disable the spel behavior - probably better to move this line there. We've also talked about adding the ability to disable spel based on a container's class, e.g.
```html
<div class="no-spel">
  <input type="text"/>
  <input type="text"/>
</div>
```
so spel could be disabled for all the elements without having to include the class on every input in a container.

My preference would be this (also prefer camel-casing the class name, i.e. `no-spel`):
```javascript
let type = el.attr('type');
if (type === 'checkbox' || type === 'radio' || el.closest('.no-spel').size()) {
  return;
}
```